cmd/qos.go

```diff
@@ -42,9 +42,15 @@ func getServiceQoSInstances(
 
 	// TODO_FUTURE: support serviceQoS-specific configuration.
 	allServiceIDs := append(gatewayConfig.GetEnabledServiceIDs(), gatewayConfig.HydratorConfig.ServiceIDs...)
+
 	for _, serviceID := range allServiceIDs {
-		switch serviceID {
-		case config.ServiceIDEVM, config.ServiceIDFEVM:
```

Comment:
Why are we deleting this from the `config` package? It seems related to configuration, i.e. which string literals can be configured to map to QoS instances.

Reply:
Sure, I have moved the map into the `config` package.